building/tracks/syllabus/first-exercise.md

```diff
@@ -0,0 +1,25 @@
+# Syllabus - The first exercise
+
+The first exercise should cover the concept `basics`, and it should have no prerequisites.
+It should provide only enough information to allow the student to solve the exercise at hand.
+
+For example, if you need to compile a program to get it to work, this is something that you'll want to mention in the introduction to the exercise.
+
+Other things that are often covered within the `basics` exercise are:
+- function or method definitions
+- calling functions
+- passing arguments to functions
+- assigning a value with a name (variables and/or constants)
+
+Use as few data types as possible.
+If you can manage, use only integers, as their notation is the same across most languages and will be well known to virtually everyone.
+
+Each language is different, and you might find that you need to introduce more than other tracks do, or less.
+Many tracks have already implemented an exercise covering `basics`.
```

Comment:
I feel it's maybe worth saying "So go and try a couple of track's first Concept Exercise (normally a variation of "Lasagna") to get a feel for what they teach".

I think encouraging people to actually go and do some other languages' exercises is a really good tip in general.

Reply:
Agreed! Fixing.